Difi.Felles.Utility.Tester/CertificateChainValidatorTests.cs

```diff
@@ -23,7 +23,7 @@ public void Fails_with_self_signed_certificate()
                 Assert.Contains("sertifikatet er selvsignert", result.Message);
             }
 
-            [Fact]
+            [Fact(Skip="Skipped because this is a windows specific task. Will deal with this in the future.")]
```

Comment:
Burde vi fikse dette før vi releaser?

Reply:
Dette er noe som blir fikset i .NET Core 3. 

https://stackoverflow.com/questions/54579288/unable-to-load-a-pem-from-a-resource-file-on-macos

https://github.com/dotnet/corefx/issues/35163

Vi kan skrive om på hvordan CertificateResource leser inn certs som en workaround, men er det en prioritet?

Reply:
Kanskje det kan vente 🤔 Kan jo ev. forklare litt mer utfyllende at det er noe som venter på en fix i .net core 3, heller enn bare "Will deal with this in the future"

Reply:
Gjort i eb2d11e